### PR TITLE
Tests

### PR DESCRIPTION
**Pull request by paulkotov**
Related to issue #1

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -21,7 +21,7 @@
     "clean": "rimraf build",
     "lint": "eslint src test",
     "prepublish": "npm run clean && npm run build",
-    "test": "mocha --compilers js:babel-register --recursive --require ./test/setup.js",
+    "test": "mocha --compilers js:babel-register --recursive ./test/components/index.jsx --require ./test/setup.js",
```
Comment (ubcent) [inline package.json]: It's just a quick fix. What if I want to test another component as a part of the whole project?

File: .eslintrc [hunk 2]
```diff
@@ -1,6 +1,7 @@
 {
   "extends": "airbnb",
   "rules": {
+    "linebreak-style" : "warn",
```
Comment (ubcent) [inline .eslintrc]: I guess you shouldn't change this config here. Let's discuss it separately

File: package.json (hunk 1 above)
Comment (paulkotov) [reply package.json]: Yes, it was rapid solvement to launch test. I believe i'll make it right.

File: .eslintrc (hunk 2 above)
Comment (paulkotov) [reply .eslintrc]: Ok

File: package.json (hunk 1 above)
Comment (ubcent) [reply package.json]: Deal

**Comment by ubcent** [comment]
By the way, @paulkotov could you please provide documentation for testing?

**Pull request merged**